src/theme/CodeBlock/Content/String.js

```diff
@@ -91,7 +91,7 @@ export default function CodeBlockString({
             />
           )}
           
-          {(metastring ?? "").includes("playground") && <button className={clsx('clean-btn')} onClick={ () => window.open(`https://play.winglang.io?code=${Buffer.from(code).toString('base64')}`) }>Play</button>}
+          {(metastring ?? "").includes("playground") && <button className={clsx('clean-btn')} onClick={ () => window.open(`https://play.winglang.io?code=(${encodeURIComponent(Buffer.from(code).toString('base64'))})`) }>Play</button>}
```

Comment:
@ekeren please check if we can remove the node polyfill i've added to support the Buffer.
(webpack 5 issue)
checkout docusaurus.config.js const NodePolyfillPlugin = require('node-polyfill-webpack-plugin');

Reply:
removed